views/js/test/runner/provider/test.js

```diff
@@ -3,8 +3,10 @@ define([
     'lodash',
     'taoItems/runner/api/itemRunner',
     'taoQtiItem/runner/provider/qti',
-    'json!taoQtiItem/test/samples/json/space-shuttle.json'
-], function($, _, itemRunner, qtiRuntimeProvider, itemData){
+    'json!taoQtiItem/test/samples/json/space-shuttle.json',
+    'json!taoQtiItem/test/samples/json/space-shuttle-PIC.json'
```

Comment:
it seems the sample is 'json!taoQtiItem/test/samples/json/space-shuttle-PIC.json'